fault/verilator_target.py

```diff
@@ -105,6 +108,14 @@ def __init__(self, circuit, directory="build/",
 
         # Save settings
         self.disp_type = disp_type
+        self.use_kratos = use_kratos
+        if use_kratos:
+            try:
+                import kratos_runtime
+            except ImportError:
+                raise ImportError("Cannot find kratos-runtime in the system. "
+                                  "Please do \"pip install kratos-runtime\" "
+                                  "to install.")
```

Comment:
This runtime import logic is repeated in the system verilog target, can we move this to a shared helper function?